torch/jit/_ir_utils.py

```diff
@@ -9,18 +8,20 @@ def __init__(
         self,
         insert_point_graph: torch._C.Graph,
         insert_point: Union[torch._C.Node, torch._C.Block],
-    ):
+    ) -> None:
         self.insert_point = insert_point
         self.g = insert_point_graph
         self.guard = None
 
-    def __enter__(self):
+    def __enter__(self) -> None:
         self.prev_insert_point = self.g.insertPoint()
         self.g.setInsertPoint(self.insert_point)
 
-    def __exit__(self, *args):
+    def __exit__(self, *args: object) -> None:
```

Comment:
This is fine - although technically the arg types are:
```
def __exit__(
        self,
        exc_type: type[BaseException] | None,
        exc_val: BaseException | None,
        exc_tb: TracebackType | None,
    ) -> None:
```

Reply:
This and the other comment are fixed in https://github.com/pytorch/pytorch/pull/143371